### ISSUE TITLE
Run Instructions

### ISSUE THREAD

**Issue by jamisonhyatt** [puniverse/capsule-demo#5]
There are no instructions for actually running the resulting capsule after the build process in the readme.md.  When following the instructions here (http://www.capsule.io/user-guide/) it says to just run "java -jar foo.jar" but that results in an exception.

When built via a "mvn package" or "gradle fatCapsule"  and then running a "java -jar foo.jar" (gradle) or "java -jar foo-1.0-capsule-fat.jar" (maven) the application fails with: CAPSULE EXCEPTION: Connection reset while processing null null: null

gradle built execution log: https://gist.github.com/hotel77/a3cc6cec9d6e0f81edf4
maven built execution log: https://gist.github.com/hotel77/ee0287ce9241a54336af

**Comment by pron** [comment]
Seems like the capsule runs fine but displays a benign exception that occurs during shutdown. 

We'll take a look.

**Comment by jamisonhyatt** [comment]
I think that is probably true for the gradle built fat jar, but it doesn't look like the maven built fat jar is working at all.

**Comment by pron** [comment]
True. It seems like a random difference, though. The problem isn't the build or the way you launch the app, but the management of the socket bewteen the capsule process and the app process.
